src/crypto/test/crypto.cpp

```diff
@@ -594,3 +595,10 @@ TEST_CASE("x509 time")
     }
   }
 }
+
+TEST_CASE("hmac")
+{
+  std::vector<uint8_t> key(0, 32);
+  std::vector<uint8_t> data(0, 64);
+  auto r = crypto::hmac(MDType::SHA256, key, data);
```

Comment:
Maybe check that calling the function twice with the same inputs return the same result, and than calling with slightly different arguments return a different output?

Reply:
Yes, done.